libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -5,6 +5,25 @@ $blur-radius: 60px;
 $max-content-width: 324px;
 $main-image-height-small: 132px;
 $main-image-height-large: 164px;
+$cq-width: 515px;
+
+@mixin cq-min() {
```

Comment:
Perfectly fine to leave the container queries in here to start with, then we can promote them to the design system at a later point 👍